src/Model/ModelManager.php

```diff
@@ -666,4 +671,20 @@ private function getValueFromType($value, Type $type, string $fieldType, Abstrac
 
         return (string) $type->convertToDatabaseValue($value, $platform);
     }
+
+    private function getFieldCustomType(?string $fieldType): ?Type
+    {
+        return Type::hasType($fieldType) ? Type::getType($fieldType) : null;
```

Comment:
Since the argument 1 for `Type::hasType()` is declared as `string`, we should not pass `null` values.

Reply:
it's generated based on the return type of `\Doctrine\ORM\Mapping\ClassMetadataInfo::getTypeOfField` as of today it's `@return string|null`

Reply:
I mean, we should respect the signature exposed by `Type::hasType()`.
This way is how it looks like at [`doctrine/dbal:2.10.2`](https://github.com/doctrine/dbal/blob/2.10.2/lib/Doctrine/DBAL/Types/Type.php#L249-L259):
```php
    /**
     * Checks if exists support for a type.
     *
     * @param string $name The name of the type.
     *
     * @return bool TRUE if type is supported; FALSE otherwise.
     */
    public static function hasType($name)
    {
        return self::getTypeRegistry()->has($name);
    }
```